libs/aws/langchain_aws/chat_models/bedrock_converse.py

```diff
@@ -334,17 +334,38 @@ class Joke(BaseModel):
     additionalModelResponseFieldPaths.
     """
 
+    supports_tool_choice_values: Optional[
+        Sequence[Literal["auto", "any", "tool"]]
+    ] = None
+    """Which types of tool_choice values the model supports.
+    
+    Inferred if not specified. Inferred as ('auto', 'any', 'tool') if a 'claude-3' 
```

Comment:
Should be claude3 and mistral large? see [this doc](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_ToolChoice.html#:~:text=Determines%20which%20tools%20the%20model%20should%20request%20in%20a%20call%20to%20Converse%20or%20ConverseStream.%20ToolChoice%20is%20only%20supported%20by%20Anthropic%20Claude%203%20models%20and%20by%20Mistral%20AI%20Mistral%20Large.)